sdk/monitor/Azure.Monitor.OpenTelemetry.AspNetCore/CHANGELOG.md

```diff
@@ -15,6 +15,12 @@
   `APPLICATIONINSIGHTS_CONNECTION_STRING` is present in `IConfiguration`.
   ([#45292](https://github.com/Azure/azure-sdk-for-net/pull/45292))
 
+* Updated field mappings for sample telemetry sent to LiveMetrics.
```

Comment:
Should this be in other section? if not then, we need more information here as its not clear what the bug fix was.

Reply:
You're right, the "Other" section is a better fit for this